modules/ROOT/pages/index.adoc

```diff
@@ -1,27 +1,13 @@
-= Bonita Intelligent Continuous Improvement Add-on
-:page-aliases: release_notes.adoc
+= Labs Program
```

Comment:
⚠️ the purpose of the alias is to redirect people from the removed release-notes page to this one instead of getting a 404 error (when no alias).
Is this what we want?

Reply:
Aliases are documented in https://docs.antora.org/antora/2.3/page/page-aliases/.
We will highlight it in the tech writer mini-guide that we are currently writing (see https://github.com/bonitasoft/bonitasoft.github.io/pull/142)

Reply:
ok thanks. Will put it back.